<ui version="4.0" >
 <class>UiInstallProcessPage</class>
 <widget class="QWizardPage" name="UiInstallProcessPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Install process</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" name="hlInstallationStatus" >
     <item>
      <widget class="QLabel" name="labelInstallationStatus" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Installation status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBarInstallation" >
       <property name="value" >
        <number>0</number>
       </property>
       <property name="textVisible" >
        <bool>true</bool>
       </property>
       <property name="invertedAppearance" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QGridLayout" name="glProcessStatus" >
     <item row="0" column="0" >
      <widget class="QLabel" name="labelProcessStatus" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>110</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Process status:</string>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="0" column="1" >
      <widget class="QListWidget" name="listWidgetDosBoxProcessStatus" >
       <property name="showDropIndicator" stdset="0" >
        <bool>false</bool>
       </property>
       <property name="dragDropMode" >
        <enum>QAbstractItemView::NoDragDrop</enum>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <spacer name="vsProcessStatus" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnInstallGame" >
       <property name="minimumSize" >
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Installation</string>
       </property>
       <property name="shortcut" >
        <string>Ctrl+I</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
